src/main/java/by/itech/lab/supplier/domain/User.java

```diff
@@ -110,7 +106,7 @@ public void setRole(Role role) {
         this.role = role;
     }
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
do you really need this data always?

Reply:
this is additional query in the database when you will load that entity (for each entity)